client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx

```diff
@@ -135,12 +135,15 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 				<>
 					<p>
 						{ translate(
-							'%(email)s is almost ready! Complete the setup to activate your new email address.',
+							'Your email {{strong}}%(email)s{{/strong}} is almost ready! Complete the setup to activate your new email address.',
```

Comment:
I'd like to suggest using `mailbox` instead of `email` at the beginning of this sentence:

> Your mailbox is almost ready! Complete the setup to activate your new email address.

That way we avoid some repetition, and we are more consistent with copy changes from D46188-code.

Reply:
Would it make sense to use mailbox instead of email in other notification messages too?

Reply:
I think it's fine to use either `mailbox` or `email address` (maybe not just `email`) in notification messages. I wouldn't worry too much about it for now though as I expect us to look into defining and using a consistent vocabulary when we implement another email provider. That's indeed something that has already caused us some confusion as developers (we also refer to those accounts as `G Suite users` a lot), I have no doubt that this preliminary work will allow us to offer a better experience to our users. Pinging @fditrapani to put that discussion on his radar.